### PR TITLE
Homework react week#1 Implemented

### PR DESCRIPTION
**Pull request by DiegoRuiz454**

### CODE REVIEW

File: react/week1/TodoApp.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ReactDOM from  'react-dom';
```
Comment (amerdahabi) [inline react/week1/TodoApp.js:1]: we only need ReactDom in our index.js file.

File: react/week1/TodoApp.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+class TodoList extends React.Component {
```
Comment (amerdahabi) [inline react/week1/TodoApp.js:3]: for better experience, you can make the TodoList in new component outside App.js , and then import it

File: react/week1/TodoApp.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  render() {
+    return [<li>{this.props.parameters.render_A} Get out of bed, {this.props.parameters.render_B} "Wed Sep 13 2017"</li>,
```
Comment (amerdahabi) [inline react/week1/TodoApp.js]: to make our code shorter and cleaner i would write this function i n a separate file like so:
```
function TodoList ({params}) {
    return (
         <div>
           <ul>
               {params.map((item)=>{
               return <li key={item.index}>{item.description} ,{item.deadline}</li>
               })}
          </ul>      
        </div>
       )
}

export default TodoList;
```

File: react/week1/TodoApp.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    const items = {render_A: "description:", render_B: "deadline date: "} ;
+    return (
+      <div>
+      <h1>Basic React & props TODO LIST</h1>
+      <TodoList parameters={items} />
+      </div>
+    );
+  }
+}
+ReactDOM.render(<Atributes />, document.getElementById('root'));
```
Comment (amerdahabi) [inline react/week1/TodoApp.js]: you can delete this line and the code will keep working, as ReactDom is in the index.js file

**Review by amerdahabi — CHANGES_REQUESTED** [review]
Nice work, i left a small notes in your code👍🏻

File: react/week1/TodoApp.js (hunk 4 above)
Comment (DiegoRuiz454) [reply react/week1/TodoApp.js]: thank you so much for the feedback amerdahabi

**Pull request merged**